Week3/homework/step2-6.js

```diff
@@ -4,13 +4,21 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  let arrContent = '';
+  for (let i = 0; i < arr.length; i++) {
+    arrContent += arr[i].join('');
+  }
+  return console.log(arrContent);
```

Comment:
There is not much point in trying to return anything from this function as the return value is unused. Besides that, you are returning the value that is returned by the console.log function. But console.log doesn't return anything (its return value is `undefined`), making this `return` statement doubly useless.